src/modules/flow/twitter/twitter.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "twitter",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "BSD-3-Clause",
+    "version": "1"
+  },
+  "types": [
+    {
+      "name": "twitter/client",
+      "category": "output/network",
+      "description": "A twitter's node client",
+      "methods": {
+        "close": "twitter_close",
+        "open": "twitter_open"
+      },
+      "options": {
+	"members": [
+	  {
+	    "data_type": "string",
+	    "description": "The consumer key (application's token)",
+	    "name": "consumer_key"
+	  },
+	  {
+	    "data_type": "string",
+	    "description": "The consumer secret key (application's secret token)",
+	    "name": "consumer_secret"
+	  }
+	],
+	"version": 1
+      },
+      "in_ports": [
+        {
+          "data_type": "string",
+          "description": "The token returned on twitter's oauth process",
+          "methods": {
+            "process": "token_process"
+          },
+          "name": "TOKEN"
+        },
+	{
+          "data_type": "string",
+          "description": "Post the string that comes into twitter",
+          "methods": {
+            "process": "post_process"
+          },
+          "name": "TWEET"
+        },
+	{
+          "data_type": "any",
+          "description": "Gets the user's timeline",
+          "methods": {
+            "process": "timeline_process"
+          },
+          "name": "GET_TIMELINE"
+        }
+      ],
+      "out_ports": [
+        {
+          "data_type": "json-object",
+          "description": "The result as json-objectt",
+          "name": "OBJECT"
+        },
+	{
+          "data_type": "json-array",
+          "description": "The result as json-array",
+          "name": "ARRAY"
```

Comment:
I want both kinds of node. We can do a `twitter/generic` that doesn't know about format, all it does is take an array or object, output an array or object.

Then we can have `twitter/tweet`, `twitter/timeline` and friends, that are easier to use.

As for the port name,s I'm okay with your names.